pkg/storaged/client.js

```diff
@@ -900,21 +903,122 @@ client.vdo_overlay = vdo_overlay();
 /* Stratis */
 
 client.stratis_start = () => {
+    return stratis2_start()
+            .catch(error => {
+                if (error.problem == "not-found")
+                    return stratis3_start();
```

Comment:
At some point we want to switch this around, but fine for now.

Reply:
Yes, agreed.